src/Component/InlineEdit.php

```diff
@@ -134,19 +134,15 @@ public function setModel(\atk4\data\Model $model)
      * onChange handler.
      * You may supply your own function to handle update.
      * The function will receive one param:
-     *  value:  the new input value.
-     *
-     * @param callable $fx
+     *  value: the new input value.
      */
-    public function onChange($fx)
+    public function onChange(\Closure $fx)
     {
-        if (is_callable($fx) && !$this->autoSave) {
-            if ($this->cb->triggered()) {
-                $value = $_POST['value'] ? $_POST['value'] : null;
-                $this->cb->set(function () use ($fx, $value) {
-                    return call_user_func($fx, $value);
-                });
-            }
+        if (!$this->autoSave) {
+            $value = $_POST['value'] ?? null;
+            $this->cb->set(function () use ($fx, $value) {
+                return call_user_func($fx, $value);
```

Comment:
`return $fx($value);`?

Reply:
I would normally write it like that, but on the other side `call_user_func*` is easier to search. But yes, lets never use call_user_func :)

Reply:
done, yes, it is better